app/docs/dev/policies/targetref.md

```diff
@@ -0,0 +1,333 @@
+Its goal is to select subsets of proxies with maximum flexibility.
+
+It looks like:
+
+```yaml
+targetRef:
+  kind: Mesh | MeshSubset | MeshService | MeshServiceSubset | MeshGatewayRoute
+  name: "my-name" # For kinds MeshService, MeshServiceSubset and MeshGatewayRoute a name can be defined
+  tags:
+    key: value # For kinds MeshServiceSubset and MeshSubset a list of matching tags can be used
+```
+
+Here's an explanation of each kinds and their scope:
+
+- Mesh: applies to all proxies running in the mesh
+- MeshSubset: same as Mesh but filters only proxies who have matching `targetRef.tags`
+- MeshService: all proxies with a tag `kuma.io/service` equal to `targetRef.name`
+- MeshServiceSubset: same as `MeshService` but further refine to proxies that have matching `targetRef.tags`
+- MeshGatewayRoute: gateway using `MeshGatewayRoute` that have a name equal to `targetRef.name`
+
+Consider the example below:
+
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: MeshAccessLog
+metadata:
+  name: example
+  namespace: {{site.default_namespace}}
+  labels:
+    kuma.io/mesh: default
+spec:
+  targetRef: # top level targetRef
+    kind: MeshService
+    name: web-frontend
+  to:
+    - targetRef: # to level targetRef
+        kind: MeshService
+        name: web-backend
+      default:
+        backends:
+          - file:
+              format:
+                plain: '{"start_time": "%START_TIME%"}'
+              path: "/tmp/logs.txt"
+  from:
+    - targetRef: # from level targetRef
+        kind: Mesh
+      default:
+        backends:
+          - file:
+              format:
+                plain: '{"start_time": "%START_TIME%"}'
+              path: "/tmp/logs.txt"
+```
+
+Using `spec.targetRef`, this policy targets all proxies that implement the service `web-frontend`.
+It defines the scope of this policy as applying to traffic either from or to `web-frontend` services.
+
+The `spec.to.targetRef` section enables logging for any traffic going to `web-backend`.
+The `spec.from.targetRef` section enables logging for any traffic coming from _any service_ in the `Mesh`.
+
+### Target resources
+
+Not every policy supports `to` and `from` levels. Additionally, not every resource can
+appear at every supported level. The specified top level resource can also affect which
+resources can appear in `to` or `from`.
+
+To help users, each policy documentation includes a table indicating which `targetRef` kinds is supported at each level.
+
+This table looks like:
+
+| `targetRef.kind`    | top level | to  | from |
+| ------------------- | --------- | --- | ---- |
+| `Mesh`              | ✅        | ✅  | ❌   |
+| `MeshSubset`        | ✅        | ❌  | ❌   |
+| `MeshService`       | ✅        | ❌  | ✅   |
+| `MeshServiceSubset` | ✅        | ❌  | ❌   |
```

Comment:
We mention MeshGatewayRoute on line [152](https://github.com/kumahq/kuma-website/pull/1131/files#diff-f25d24e526ca4764872874b844778d34817a88616a050f4962aa713fc50e5c08R152) but not here, should we make this consistent?